src/main/java/cam72cam/immersiverailroading/util/RecipeUtil.java

```diff
@@ -0,0 +1,13 @@
+package cam72cam.immersiverailroading.util;
+
+import cam72cam.immersiverailroading.ImmersiveRailroading;
+import net.minecraft.item.crafting.IRecipe;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.registries.IForgeRegistryModifiable;
+
+public class RecipeUtil {
```

Comment:
This should not need to be it's own class/function.  Please collapse this functionallity into registerRecipes